client/src/Components/Navigation/Menu.js

```diff
@@ -1,101 +1,158 @@
-                    <a href="index3.html" className="brand-link">
-                        <img src="dist/img/AdminLTELogo.png" alt="AdminLTE Logo" className="brand-image img-circle elevation-3" style={{ opacity: '.8' }} />
-                        <span className="brand-text font-weight-light">TRRADER</span>
-                    </a>
-                    {/* Sidebar */}
-                    <div className="sidebar">
-                        {/* Sidebar user panel (optional) */}
-                        <div className="user-panel mt-3 pb-3 mb-3 d-flex">
-                            <div className="image">
-                                <img src="dist/img/user2-160x160.jpg" className="img-circle elevation-2" alt="User Image" />
-                            </div>
-                            <div className="info">
-                                <a href="#" className="d-block">User</a>
-                            </div>
-                        </div>
-                        {/* SidebarSearch Form */}
-                        <div className="form-inline">
-                            <div className="input-group" data-widget="sidebar-search">
-                                <input className="form-control form-control-sidebar" type="search" placeholder="Search" aria-label="Search" />
-                                <div className="input-group-append">
-                                    <button className="btn btn-sidebar">
-                                        <i className="fas fa-search fa-fw" />
-                                    </button>
-                                </div>
-                            </div>
-                        </div>
-                        {/* Sidebar Menu */}
-                        <nav className="mt-2">
-                            <ul className="nav nav-pills nav-sidebar flex-column" data-widget="treeview" role="menu" data-accordion="false">
-                                {/* Add icons to the links using the .nav-icon class
+const Menu = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const location = useLocation();
+  const [user, setUser] = useState(JSON.parse(localStorage.getItem("profile")));
+  useEffect(() => {
+    const token = user?.token;
+    if (token) {
+      const decodedToken = decode(token);
+      if (decodedToken.exp * 1000 < new Date().getTime()) logout();
+    }
+    setUser(JSON.parse(localStorage.getItem("profile")));
+  }, [location]);
+  const logout = () => {
+    dispatch({ type: "LOGOUT" });
+    history.push("/");
+    setUser(null);
+  };
+  return (
+    <div>
+      <aside className="main-sidebar sidebar-dark-primary elevation-4">
+        {/* Brand Logo */}
+        <a href="index3.html" className="brand-link">
+          <img
+            src="dist/img/AdminLTELogo.png"
+            alt="AdminLTE Logo"
+            className="brand-image img-circle elevation-3"
+            style={{ opacity: ".8" }}
```

Comment:
it would be great to have this be in a class instead

Reply:
is there a reason this is happening as inline styling instead of in a css class?

Reply:
I left it like that because I was following an instructional video on how to implement the template. 
I can work with Sona to change the css to make it it's own class.